### ISSUE TITLE
Add support for non auto-increment/integer primary keys

### ISSUE THREAD

**Issue by mikeywaites** [rbarrois/factory_boy#92]
Our system makes use of UUID column type in postgres in some places.  At the moment FactoryBoy will try to perform a lookup on the primary key  to determine the next sequence using the max() function on the primary key column of a model

```
/factory/alchemy.py", line 38, in _setup_next_sequence
max_pk = session.query(max(pk)).one()[0]

#produces the following sql statement.
SELECT max(foo.id) AS max_1 \nFROM foo' {}
```

This is obviously not going to work when we have UUID's 

Is this something you might consider allowing for?

**Comment by rightaway** [comment]
I've just tested with SQLAlchemy and can confirm 13d310f removes the need for the workaround I had before. Great!

```
@classmethod
def _setup_next_sequence(cls, *args, **kwargs):
    return 0
```

**Comment by rbarrois** [comment]
This issue has been fixed in 13d310f, and will be part of the 2.5.0 release.
